src/Npgsql.GeoJSON/Internal/CrsMap.cs

```diff
@@ -21,7 +21,7 @@ internal CrsMapEntry(int minSrid, int maxSrid, string? authority)
     }
 }
 
-ref struct CrsMapBuilder
+struct CrsMapBuilder
```

Comment:
Upgrading to the rc1 SDK started throwing:

```
/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Internal/CrsMap.cs(47,21): error CS8156: An expression cannot be used in this context because it may not be passed or returned by reference [/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj]
/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Internal/CrsMap.cs(47,13): error CS8350: This combination of arguments to 'CrsMapBuilder.AddCore(in CrsMapEntry)' is disallowed because it may expose variables referenced by parameter 'entry' outside of their declaration scope [/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj]
/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Internal/GeoJSONTypeHandlerResolver.cs(40,37): error CS8156: An expression cannot be used in this context because it may not be passed or returned by reference [/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj]
/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Internal/GeoJSONTypeHandlerResolver.cs(40,25): error CS8350: This combination of arguments to 'CrsMapBuilder.Add(in CrsMapEntry)' is disallowed because it may expose variables referenced by parameter 'entry' outside of their declaration scope [/home/roji/projects/npgsql/src/Npgsql.GeoJSON/Npgsql.GeoJSON.csproj]
```

In any case, there's no reason to use a ref struct here, so simplifying.